spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -788,7 +788,13 @@ def testers_by_app(tester, app_id)
       parse_response(r, 'data')['users']
     end
 
-    def create_tester!(tester: nil, email: nil, first_name: nil, last_name: nil)
+    def groups
+      return @cached_groups if @cached_groups
+      r = request(:get, '/WebObjects/iTunesConnect.woa/ra/users/pre/ext')
+      @cached_groups ||= parse_response(r, 'data')['groups']
```

Comment:
The `||=` is unnecessary here since you're checking `@cached_groups` for existence 2 lines above